sdk/test/IntegrationTests/Tests/General.cs

```diff
@@ -659,6 +661,7 @@ private static void TestCredentials(ProxyRefreshingAWSCredentials creds, bool ex
                 {
                     client.ListBuckets();
                     Assert.IsFalse(expectFailure);
+                    Assert.IsNotNull(creds.Expiration);

```

Comment:
`ProxyRefreshingAWSCredentials` (the type for this `creds` object) extends `RefreshingAWSCredentials` so I'm verifying the property was populated correctly.